package.json

```diff
@@ -10,8 +10,8 @@
     "lint:check": "eslint . --ignore-path .gitignore",
     "format": "prettier . --write --ignore-path ./.gitignore",
     "format:check": "prettier . --check --ignore-path ./.gitignore",
-    "storybook": "start-storybook -p 6006 --no-dll",
-    "storybook:build": "build-storybook --no-dll",
+    "storybook": "start-storybook -p 6006",
+    "storybook:build": "build-storybook",
```

Comment:
Good to see they got rid of those precompiled dlls! 👍 
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecated-dll-flags